src/list/list.service.ts

```diff
@@ -0,0 +1,50 @@
+import { v4 as uuidv4 } from 'uuid';
+import { addedData201, internalServerError500, OK200 } from '../constants';
+import { getListOfContacts } from '../contact/contact.utils';
+import { ContactsList } from '../contactsList/contactsList.model';
+import { WorkWithDB } from '../interfaces';
+import { showData } from '../utils/utils';
+import { List } from './list.model';
+
+export const createNewList: WorkWithDB = async (req) => {
+  const { name } = JSON.parse(req.body.list) as { name: string };
+  try {
+    const res = await List.create({
+      uuid: uuidv4(),
+      name,
+    });
+    return addedData201;
+  } catch (err) {
+    return internalServerError500;
+  }
+};
+
+export const getContactsOfList: WorkWithDB = async (req) => {
+  const listId: string = req.params.listId;
+  try {
+    const res = await ContactsList.findAll({
+      attributes: ['contactId'],
+      where: { listId },
+    });
+    const answer = await getListOfContacts(res);
```

Comment:
The task was to create relations between List model and Contacts model. You should use https://sequelize.org/master/manual/assocs.html

Reply:
Changed. Utilized features of sequelize.